### PR TITLE
Created logic for build SWC-registry-python package

### PR DESCRIPTION
**Pull request by ersul4ik**
Main file is - get_entry_info.py 
This file consists of few methods:

-  get_content. The method is used to get `content` of the current SWC_ID.
- get_title. The method is used to get `title` of the current SWC_ID.
- get_relationships. The method is used to get `relationships` of the current SWC_ID.
- get_description. The method is used to get `description` of the current SWC_ID.
- get_remediation. The method is used to get `remediation` of the current SWC_ID.

Also a class (SWC) has a two methods:
- get_last_version. The method is used to get last version of swc-definition.json file. 
- get_content_by_file. The method is used to get content of swc-definition.json file out localStorage

### CODE REVIEW

File: swcregistrypython/get_entry_info.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    def __init__(self, swc_id):
+        self.swc_id = swc_id
+        self.path_file_content = 'swc-definition.json'
+
+    def get_content_by_file(self):
+        with open(self.path_file_content, 'r') as f:
+            response = json.load(f)
+        return response
+
+    def get_last_version(self):
```
Comment (JoranHonig) [inline swcregistrypython/get_entry_info.py:17]: Shouldn't this be private?

File: swcregistrypython/get_entry_info.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        try:
+            url = ('https://raw.githubusercontent.com/SmartContractSecurity/SWC-registry/master/export/swc-definition.json')
+            response = requests.get(url).json()
+            with open(self.path_file_content, 'w') as outputfile:
+                json.dump(response, outputfile)
+        except requests.exceptions.RequestException:
+            response = self.get_content_by_file()
+        return response
+
+    def get_content(self):
```
Comment (JoranHonig) [inline swcregistrypython/get_entry_info.py]: I'd make the function:
```python3
@property
def content(self):
   ...
```

That allows me to do:
```python3
swc = SWC('SWC-100')
print(swc.title)
```

Same for the other properties

File: swcregistrypython/get_entry_info.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import requests
+import json
+
+
+class SWC:
+    def __init__(self, swc_id):
+        self.swc_id = swc_id
+        self.path_file_content = 'swc-definition.json'
+
+    def get_content_by_file(self):
```
Comment (JoranHonig) [inline swcregistrypython/get_entry_info.py:11]: Shouldn't this be private?
```suggestion
    def _get_content_by_file(self):
```

**Comment by JoranHonig** [comment]
Hi,

So one thing that I wanted to see is that instead of getting the swc entries through a webrequest that these are part of the package itself. Right now it does both.
I want to be able to specify an option to not use the web to get the definition swc. Earlier in mythril-classic users have expressed the concern that they don't want mythril classic to make web requests when not allowed to.
Secondly, I don't really se a reason to add web requests. It has the negative effect that even though I'm using the same version of the package the behavior of the package can change (for example on an edit or removal of an entry). I'd rather have that the definition json is only in the package itself and updated on a package update.

Besides that (maybe not in this pr ;)) I think it would be a good idea to setup a read the docs/ or documentation section in the current swc github pages, that describes the libary and its provided functions. (You can very well use pydoc and create such a documentation page from that)
Another element in the documentation category would be the addition of examples. I see that right now the library is very simple, but I think that an example or 2 would be able to instantly convey how the library should be used.

One minor detail: I see that the author_email in setup.py is set to your personal email. It might be a good idea to setup an organization email address for SmartContractSecurity

The code and content of this pr itself looks quite alright! Great job :+1: Only minor comments

Cheers!

**Comment by thec00n** [comment]
@JoranHonig swc-definitions.json come packaged by default and you can choose to update them through `get_last_version()` if you want. Agree on documentation and adding that part of another PR.

**Pull request merged**